taskfiles/lint.yaml

```
@@ -750,6 +753,9 @@ tasks:
           # yamllint enable rule:line-length
           FLAGS:
             - "--config-file '{{.ROOT_DIR}}/.clang-tidy'"
+            # The following line filter is added to ignore msgpack issue #1098:
+            # https://github.com/msgpack/msgpack-c/issues/1098
+            - "-line-filter \"[{\\\"name\\\":\\\"cpp11_zone.hpp\\\",\\\"lines\\\":[[197,197]]}]\""
```

```suggestion
            - >-
              --line-filter "[{\"name\":\"cpp11_zone.hpp\",\"lines\":[[197,197]]}]"
```

* So it's more readable.
* Probably a good idea to extract it into a variable rather than duplicating it, right?